spec/controllers/application_templates_controller_spec.rb

```diff
@@ -173,4 +173,54 @@
       end
     end
   end
+
+  describe 'POST #toggle_unavailability_enabled' do
+    before :each do
+      department = create :department, name: 'Bus'
```

Comment:
The specific department name and specific position name don't really matter for the test, right/